src/main/java/net/sf/marineapi/ais/message/AISUTCReport.java

```diff
@@ -25,7 +25,7 @@
  * 
  * @author Lázár József
  */
-public interface AISUTCReport extends AISMessage {
+public interface AISUTCReport extends AISMessage, AISPositionInfo {
```

Comment:
ATM, there may not be UTC messages without the position, but I think it would still be safer design to keep the UTC and position interfaces separate, thus allowing to use them independently.

Reply:
Ok, I'll change it back. Did this because the actual fields are in the UTC message implementation.